packages/gatsby/src/query/index.ts

```diff
@@ -45,16 +56,27 @@ function calcDirtyQueryIds(state) {
   return dirtyQueryIds
 }
 
+// TODO: Possibly remove? Used in inc-build-cli
+export function calcInitialDirtyQueryIds(state: IGatsbyState): Array<string> {
```

Comment:
This only exists because the inc-build-cli test calls this :O

Reply:
I think we will keep it for compatibility (better not to break it in any minor). 

That said I think
```js
export { calcDirtyQueryIds as calcInitialDirtyQueryIds }
```
would maintain semantics of what we have today
```js
module.exports = {
  calcInitialDirtyQueryIds: calcDirtyQueryIds,
  // rest of exports
}
```
 and will be shorter/easier to grep that this is just exporting same thing under different export name